dvc/analytics.py

```diff
@@ -161,8 +161,8 @@ def collect(self):
         """Collect analytics report."""
         from dvc.scm import SCM
         from dvc.utils import is_binary
-        from dvc.project import Project
-        from dvc.exceptions import NotDvcProjectError
+        from dvc.repo import Repo
```

Comment:
:see_no_evil: hate to be that one... but why not "Repository" ?

`libgit2` and `git` are going with the full word:
- https://github.com/libgit2/libgit2/blob/635693d3bc55770ec7a6640ba3f2f0ee434a6042/src/repository.h#L129
- https://github.com/git/git/blob/master/repository.h#L14

I know it is shorter but I think it makes more sense.

_NOTE: I was hesitating to post this one, I know it would be really painful to change it, as it goes more than a `sed -i s/repo/repository/g` (or a more thoughtful regex), so don't take it seriously, @efiop_ :slightly_smiling_face:

Reply:
@mroutis I was actually using gitpython's Repo as an example. Seems short and sweet.

Reply:
@mroutis No logic changes so far. I'm about to push a separate commit here to reorganize metrics logic though. This PR is more of a first step, just to separate everything so it is easier for everyone to work on top of it after that.